src/Utilities/BlockParser.f90

```diff
@@ -583,8 +584,7 @@ subroutine DevOpt(this)
     if (IDEVELOPMODE == 0) then
       errmsg = "Invalid keyword '"//trim(this%laststring)// &
                "' detected in block '"//trim(this%blockname)//"'."
-      call store_error(errmsg)
-      call this%StoreErrorUnit()
+      call dev_feature(errmsg, this%iuext)
```

Comment:
Nice idea to break this out.  I see that we are double-checking IDEVELOPMODE both in parser%DevOpt() and in dev_feature().  Maybe we can move that entirely into dev_feature()?

Reply:
Removed IDEVELOPMODE usage from BlockParser. Thanks, I overlooked that.